rubocop-rspec_rails.gemspec

```diff
@@ -34,5 +34,6 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
+  spec.add_runtime_dependency 'rack'
```

Comment:
I think we should only add this dependency once we have released a few versions of rubocop-rspec that don’t depend on rubocop-rspec_rails.